### ISSUE TITLE
checkbox tabbing

### ISSUE THREAD

**Issue by avalanche1** [Semantic-Org/Semantic-UI#2254]
What's the point of setting ```tabindex=0``` after toggling checkbox?
```javascript
set: {
          checked: function() {
            $module.addClass(className.checked);
          },
          tab: function() {
            if( $input.attr('tabindex') === undefined) {
              $input
                .attr('tabindex', 0)
              ;
            }
          }
        },
```
I, for one, would like to check and uncheck it using ```space```, but instead form navigation focus gets reset to nowhere after pressing ```space``` for the first time.

**Comment by jlukic** [comment]
It doesnt set tabindex after toggling, it sets it on init.

**Comment by avalanche1** [comment]
uhm... ok, maybe I read it wrong, but here is a fiddle that demonstrates that after pressing ```space``` focus gets blurred to nowhere, while it should remain on the checkbox
http://jsfiddle.net/efp8z6Ln/149/
And also it would be sensible that if I click the checkbox, it will get focused.

**Comment by jlukic** [comment]
There was a bug but it was not related to what you mentioned. `$input.blur()` was being called on checkbox `toggle`, this was added as a fix a long time ago #103 for form validation not triggering validation on change, but is no longer necessary.

**Comment by avalanche1** [comment]
awesome! thank you. will be waiting for rebuild of ```next``` to check it out :smile:
